### PR TITLE
GCM and minor DX12 fixes

### PR DESCRIPTION
**Pull request by tambry**
Fixes the main menu colours in Terraria - Trial, but character creation menu's colours on the character's display worsened.

Also adds AlexAltea to mention-bot blacklist.

### CODE REVIEW

File: rpcs3/Emu/SysCalls/Modules/cellGcmSys.cpp [hunk 1]
```diff
@@ -560,7 +560,7 @@ s32 cellGcmSetTileInfo(u8 index, u8 location, u32 offset, u32 size, u32 pitch, u
 	cellGcmSys.Warning("cellGcmSetTileInfo(index=%d, location=%d, offset=%d, size=%d, pitch=%d, comp=%d, base=%d, bank=%d)",
 		index, location, offset, size, pitch, comp, base, bank);
 
-	if (index >= rsx::limits::tiles_count || base >= 800 || bank >= 4)
```
Comment (Nekotekina) [inline rpcs3/Emu/SysCalls/Modules/cellGcmSys.cpp]: Funny bug, it should have been 0x800 (which is equal to 2048, just in case)

**Comment by tambry** [comment]
@vlj Could you check out the Terraria fix and see if it's OK to be merged?

**Comment by jarveson** [comment]
@tambry It looks like my rsx/swizzle pr may have fixed the terraria colors issue. It hangs for me trying to get to character creation (i think from savedata), but main menu colors are correct. Also, that 'is_swizzled' flag is the flag for morton/zordering and not color channel swizzling, so if there is still a color issue, it may want to be solved differently

**Comment by tambry** [comment]
@jarveson Seems like it did fix it. I have removed the "fix" commit.

**Pull request merged**